docs/features/files_and_mounts.md

```diff
@@ -67,7 +71,27 @@ It's important to notice that, when copying the directory to the container, the
 
 You can leverage the very same mechanism used for copying files to a container, but for directories.:
 
-1. The first way is using the `Files` field in the `ContainerRequest` struct, as shown in the previous section, but using the path of a directory as `HostFilePath`.
+1. The first way is using the `Files` field in the `ContainerRequest` struct, as shown in the previous section, but using the path of a directory as `HostFilePath`. Like so:
+
+```go
+ctx := context.Background()
```

Comment:
If possible, this code snippet should come from an embedded test, so that the snippet does not get outdated. I think it relates to #2202